src/v2/Components/NavBar/Menus/DropDownMenu/DropDownMenu.tsx

```diff
@@ -5,10 +5,37 @@ import React from "react"
 import styled from "styled-components"
 import { DropDownSection } from "./DropDownSection"
 import { Menu, MenuItem } from "v2/Components/Menu"
+import { MenuData } from "../../menuData"
+
+const VIEW_ALL_TOP_MARGINS = {
+  HeaderArtworksDropdown: "120px",
+  HeaderArtistsDropdown: "80px",
```

Comment:
So, we're using the ContextModule as a trigger for styles. I'll just fix the CSS to do this without hardcoded values. Both using an analytics thing for styling, and hardcoded values for positioning should be avoided.